bin/pull-prod-admin-config

```diff
@@ -0,0 +1,147 @@
+#! /bin/bash
```

Comment:
Is this largely a copy of breakglass?  Can we modularize that at all so that we're not duplicating complicated scripts?

Reply:
I'm open to thoughts on this. Yes, it feels gross to duplicate a bunch of code like this instead of factoring it out. OTOH the pull code here is very specific to a single deployment and I'm hesitant to invest the time in refactoring it. I also generally err to the side of reducing duplication once it gets to three rather than two instances.

Reply:
wrt duplication: it seems like we have no good way to manage that "3rd times the charm" part though....  and seemingly we're already modifying one of the forks with the style changes here, so it's already kinda impossible to dedup them when a 3rd need comes along.

the "specific to a single deployment" part is also kinda messy right since that's not clear either.. but is implicit in the nature of the code base.

What's a good way to not get bogged down but leave it better than we found it?  Make a bin/seattle dir with this there so at least it's more isolated from the larger ideally agnostic code base?

Reply:
> What's a good way to not get bogged down but leave it better than we found it? Make a bin/seattle dir with this there so at least it's more isolated from the larger ideally agnostic code base?

That sounds good to me. I would like our shell scripts to meet our quality bar, but I don't think refactoring this stuff is the best use of time right now. Admittedly it would probably not be such a time concern if I were better at working with shell scripts.

Another issue is that I can't test these scripts without running them in cloud shell which would make refactoring even slower.